skype/skype-ps/skype/Grant-CsCallingLineIdentity.md

```diff
@@ -5,85 +5,72 @@ applicable: Microsoft Teams, Skype for Business Online
 title: Grant-CsCallingLineIdentity
 schema: 2.0.0
 manager: bulenteg
-author: tomkau
-ms.author: tomkau
+author: jenstrier
+ms.author: jenstr
 ms.reviewer:
 ---
 
 # Grant-CsCallingLineIdentity
 
 ## SYNOPSIS
-Use the `Grant-CsCallingLineIdentity` cmdlet to apply a Caller ID policy to a user account.
+Use the `Grant-CsCallingLineIdentity` cmdlet to apply a Caller ID policy to a user account or set the tenant Global instance.
 
 ## SYNTAX
 
+### GrantToTenant (Default)
 ```
-Grant-CsCallingLineIdentity [[-Identity] <UserIdParameter>] [[-PolicyName] <String>] [-Confirm] [-DomainController <Fqdn>] [-PassThru] [-Tenant <Guid>] [-WhatIf] [<CommonParameters>]
+Grant-CsCallingLineIdentity [[-PolicyName] <string>] [-Global] [-PassThru] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+### Identity
+```
+Grant-CsCallingLineIdentity [[-Identity] <string>] [[-PolicyName] <string>] [-PassThru] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-You can either change or block the Caller ID (also called a Calling Line ID) for a user. By default, the user's phone number can be seen when that user makes a call to a PSTN phone, or when a call comes in. You can create a Caller ID policy to provide an alternate displayed number, or to block any number from being displayed; you can then apply the Caller ID policy to a specific user.
+You can either a Caller ID policy to a specific user or you can set the Global policy instance.
```

Comment:
```suggestion
You can either create a Caller ID policy for a specific user or you can set the Global policy instance.
```

Reply:
@JensTrier, I am not sure about this. Please feel free to change it. However, the original line is incomplete.